packages/investor-foxy/src/api/api.ts

```diff
@@ -455,25 +447,22 @@ export class FoxyApi {
     if (!amountDesired.gt(0)) throw new Error('Must send valid amount')
     if (!wallet) throw new Error('Missing inputs')
 
-    let estimatedGasBN: BigNumber
+    let estimatedGas: string
     try {
-      estimatedGasBN = await this.estimateDepositGas(input)
+      estimatedGas = await this.estimateDepositGas(input)
     } catch (e) {
       throw new Error(`Estimate Gas Error: ${e}`)
     }
 
     const stakingContract = this.getStakingContract(contractAddress)
-    const userChecksum = this.web3.utils.toChecksumAddress(userAddress)
+    // const userChecksum = ethers.utils.getAddress(userAddress)
```

Comment:
Remove me  - 0-value and `from` as a checksummed version of the caller address are implied as confirmed by the broadcasted Txs on-chain, calldata is the exact same with or without these `value` and `from` params